lib/public/views/Runs/Details/runDetailsComponent.js

```diff
@@ -105,6 +109,7 @@ export const runDetailsComponent = (runDetailsModel, router) => runDetailsModel.
                     epnInfologgerLinkUrl && h('a', { href: epnInfologgerLinkUrl }, 'EPN'),
                     h('', '-'),
                     qcGuiLinkComponent(run),
+                    isRunConsideredRunning(run) && aliEcsEnvironmentLink && [h('', '-'), aliEcsEnvironmentLink],
```

Comment:
I think that we should try to be consistent, and have separated things for
- getting URL
- link component
Here for example, the links do not open links in a new tab (i don't know if it's a good thing or not).
If you think using the component is the best thing here, please update the infologger URLS to be consistents.

Reply:
all links have `target: '_blank'` so they open it

Reply:
refactored